src/client/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -313,6 +335,25 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter {
             }
         });
 
+        if ((message as DebugProtocol.StackTraceResponse).command === 'stackTrace') {
+            (message as DebugProtocol.StackTraceResponse).body.stackFrames.forEach((sf) => {
+                // Check if we're stopped at the last line
+                let currentCell: NotebookCell | undefined;
+                this.notebookDocument.getCells().forEach((cell) => {
+                    const index = sf.source?.path!.indexOf('#ch');
```

Comment:
this should be a helper function, give a stack we should find the corresponding cell.
I'm assuming we have similar code today elsewhere.
Also no idea what `#ch` is here.

Reply:
I'll put it on a helper. The `#ch` is to find the fragment in the stackFrame